nio_cli/commands/blockcheck.py

```diff
@@ -0,0 +1,241 @@
+import json
+import os
+import sys
+import subprocess
+
+from .base import Base
+
+
+class BlockCheck(Base):
+
+    # This should be run from inside the block dir
+
+    def __init__(self, options, *args, **kwargs):
+
+        super().__init__(options, *args, **kwargs)
+        self._block = os.getcwd().split('/')[-1]
+        self.all_contents = os.listdir('.')
+        self.block_files_including_base = [
```

Comment:
Instead of creating this property, just pass `all_contents` to `get_versions_and_class_names`